flyteadmin/pkg/repositories/gormimpl/task_repo.go

```diff
@@ -30,12 +30,12 @@ func (r *TaskRepo) Create(ctx context.Context, input models.Task, descriptionEnt
 			}
 			return nil
 		}
-		tx := r.db.WithContext(ctx).Omit("id").Create(descriptionEntity)
+		tx := r.db.WithContext(ctx).Omit("id").Create(&input)
```

Comment:
I think this change is technically not necessary since this is all wrapped in a transaction, and if any insert fails then the whole transaction should be rolled back.